.github/workflows/windows.yml

```diff
@@ -46,15 +46,13 @@ jobs:
       - name: pip-packages
         run: |
           pip install numpy==1.22.2 cython pybind11 pythran meson ninja pytest pytest-xdist pytest-timeout pooch rich_click click doit pydevtool
-      - name: openblas-libs
+      - name: Install OpenBlas
+        shell: bash
         run: |
-          # Download and install pre-built OpenBLAS library
-          # Built with mingw-w64, -ucrt -static.
-          # https://github.com/matthew-brett/openblas-libs/blob/ucrt-build/build_openblas.ps1
-          choco install wget unzip -y --no-progress
-          wget -q https://anaconda.org/multibuild-wheels-staging/openblas-libs/v0.3.20-571-g3dec11c6/download/openblas-v0.3.20-571-g3dec11c6-win_amd64-gcc_10_3_0.zip
-          unzip -d c:\opt openblas-v0.3.20-571-g3dec11c6-win_amd64-gcc_10_3_0.zip
-          echo "PKG_CONFIG_PATH=c:\opt\64\lib\pkgconfig;" >> $env:GITHUB_ENV
+          # same OpenBLAS install method as cibuildwheel
```

Comment:
To my mind this is an easier/standard way to install OpenBLAS. Or we could use the openblas_support.py script directly.

Reply:
Yes, agreed. That custom build is no longer needed.

Reply:
I wouldn't call `openblas_support.py` "standard" though, it's an unfortunate necessity to have that around due to the limitations of PyPI. It's for our usage only; no user, contributors or downstream distributor should be using that. And I hope we'll get rid of it completely at some point, replaced by `pip install openblas` (that's in the works).

Reply:
That'll be an awesome day